documentation/modules/appendix_crds.adoc

```diff
@@ -1453,6 +1455,51 @@ Used in: xref:type-BrokerCapacity-{context}[`BrokerCapacity`]
 |string
 |====
 
+[id='type-ApiUser-{context}']
+= `ApiUser` schema reference
+
+Used in: xref:type-CruiseControlSpec-{context}[`CruiseControlSpec`]
+
+
+[options="header"]
+|====
+|Property         |Description
+|name      1.2+<.<a|Cruise Control REST API user. The following names are reserved for Strimzi and cannot be used: admin, healthcheck.
+|string
+|password  1.2+<.<a|Specify the password for the user Cruise Control REST API user.
+|xref:type-Password-{context}[`Password`]
+|role      1.2+<.<a|Cruise Control REST API user role.Valid API user roles are VIEWER, USER, and ADMINFor more information on valid API user roles see https://github.com/linkedin/cruise-control/wiki/Security#authorization.
```

Comment:
Spaces between the sentences? Do we need the link here? Certainly in the docs, we don't need it as we reference it earlier in the section. Maybe it's good to have it in the yaml etc.